src/ansys/fluent/core/launcher/container_launcher.py

```diff
@@ -80,12 +82,14 @@ def __init__(
         product_version : FluentVersion, optional
             Version of Ansys Fluent to launch. Use ``FluentVersion.v241`` for 2024 R1.
             The default is ``None``, in which case the newest installed version is used.
-        version : str, optional
+        dimension : Dimension or str, optional
             Geometric dimensionality of the Fluent simulation. The default is ``None``,
-            in which case ``"3d"`` is used. Options are ``"3d"`` and ``"2d"``.
-        precision : str, optional
+            in which case ``"three"`` is used. Options are either the values of the ``Dimension``
+            enum or any of ``"three"`` and ``"two"``.
```

Comment:
I'm wondering why we're saying "three" now for the string counterpart. Is it relevant - isn't it still "3d"? I also thing thath the documentation should lead with the enum and not the string version. The enum is the _proper_ input.

Reply:
It should be either "two" and "three" or the enums.

Reply:
I think it makes more sense as to dimension="two" rather than dimension="2d", right?

Reply:
Ideally, the type should be enum or int.

Reply:
Done, Thank you.
Now it supports enum and int.